app/domain/logs.rb

```diff
@@ -29,50 +29,58 @@ module Security
 
       end
 
-      module AuthnOidc
-
-        ExtractedUsernameFromIDToked = ::Util::TrackableLogMessageClass.new(
-          msg: "Extracted username '{0}' from ID Token field '{1-id-token-username-field}'",
-          code: "CONJ00004D"
-        )
+      module OAuth
 
-        TokenDecodeSuccess = ::Util::TrackableLogMessageClass.new(
-          msg: "Token decode succeeded",
-          code: "CONJ00005D"
-        )
-
-        TokenDecodeFailed = ::Util::TrackableLogMessageClass.new(
-          msg: "Failed to decode the token with the error '{0-exception}'",
-          code: "CONJ00018D"
-        )
-
-        OIDCProviderUri = ::Util::TrackableLogMessageClass.new(
+        IdentityProviderUri = ::Util::TrackableLogMessageClass.new(
           msg: "Working with Identity Provider {0-provider-uri}",
           code: "CONJ00007D"
         )
 
-        OIDCProviderDiscoverySuccess = ::Util::TrackableLogMessageClass.new(
+        IdentityProviderDiscoverySuccess = ::Util::TrackableLogMessageClass.new(
           msg: "Identity Provider discovery succeeded",
           code: "CONJ00008D"
         )
 
+        IdentityProviderCertificateFetchedFromCache = ::Util::TrackableLogMessageClass.new(
+          msg: "Identity Provider certificates fetched successfully from cache",
```

Comment:
In OIDC it is certificates but in Azure they are keys. Let's think of a more general word instead of ` certificates `

Reply:
I'll go with keys